config/routes.rb

```diff
@@ -127,7 +127,16 @@
       get "confirmation", on: :collection
     end
   end
-  resources :consent_forms, only: [:show]
+  resources :consent_forms, only: [:show] do
```

Comment:
We should avoid underlines in URLs:

```suggestion
  resources :consent_forms, path: "consent-forms", only: [:show] do
```

Reply:
good point - fixed